app/src/main/AndroidManifest.xml

```diff
@@ -8,6 +8,7 @@
     <uses-permission android:name="android.permission.CAMERA" />
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
Is this needed?

Reply:
Oh this is a mistake. There shouldn't the same permission twice. I will fix this.